### ISSUE TITLE
Documentation on Error codes is missing

### ISSUE THREAD

**Issue by dtrahan41** [ResearchComputing/Research-Computing-User-Tutorials#18]
Need to add information on Slurm error codes. In particular: QOSGrpCpuLimit and QOSGrpNodeLimit

**Comment by daschaich** [comment]
Yes, this would be quite useful.  I'm trying to start running on Summit, and everything I submit is being flagged with `QOSGrpCpuLimit`.  I've seen one of these jobs switch between `Priority` and `QOSGrpCpuLimit`, so I'm guessing that it will eventually run once other folks' jobs being charged to the same allocation finish, but this is just guesswork on my part---a few hours of Googling have not provided a clear answer.
